### PR TITLE
New rules from 03B236B23B1EC37C663527C1F53AF3FE

### PR DESCRIPTION
**Pull request by mike-hunhoff**
Adding new rules identified in sample with MD5 hash 03B236B23B1EC37C663527C1F53AF3FE.

### CODE REVIEW

File: data-manipulation/encryption/encrypt-or-decrypt-via-wincrypt.yml [hunk 1]
```diff
@@ -10,10 +10,11 @@ rule:
...
-        - api: CryptGenKey
-        - api: CryptImportKey
       - or:
         - api: CryptEncrypt
         - api: CryptDecrypt
+      - optional:
+        - or:
+          - api: CryptAcquireContext
+          - api: CryptGenKey
+          - api: CryptImportKey
```
Comment (mike-hunhoff) [inline data-manipulation/encryption/encrypt-or-decrypt-via-wincrypt.yml:20]: Making this block optional so the rule can match samples that call these APIs in a different function.

File: host-interaction/session/get-session-integrity-level.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+rule:
+  meta:
+    name: get session integrity level
+    namespace: host-interaction/session
```
Comment (mike-hunhoff) [inline host-interaction/session/get-session-integrity-level.yml:4]: This might not be the best name or correct location. Open to alternatives.

**Comment by mike-hunhoff** [comment]
Sample binary for these rules in PR https://github.com/fireeye/capa-testfiles/pull/7.

File: data-manipulation/encryption/encrypt-or-decrypt-via-wincrypt.yml (hunk 1 above)
Comment (williballenthin) [reply data-manipulation/encryption/encrypt-or-decrypt-via-wincrypt.yml:20]: nice, i encountered the same issue this week

**Pull request merged**